examples/syn-flood/main.rs

```diff
@@ -45,16 +47,23 @@ fn install(qs: HashMap<String, PortQueue>) -> impl Pipeline {
     .send(qs["eth1"].clone())
 }
 
+fn print_stats() {
+    let mut observer = YamlBuilder::new().build();
```

Comment:
did I miss the Yaml?

Reply:
or is it something hidden?

Reply:
ah, it's the output. fine. lame. isn't json better then haha.

Reply:
yaml hater